aws-rds-dbcluster/aws-rds-dbcluster.json

```diff
@@ -175,8 +174,7 @@
     },
     "MonitoringInterval": {
       "description": "The interval, in seconds, between points when Enhanced Monitoring metrics are collected for the DB cluster. To turn off collecting Enhanced Monitoring metrics, specify 0. The default is 0.",
-      "type": "integer",
-      "default": 0
```

Comment:
I think we'd need to remove the default for DBInstance as well, since the CLI defaults to 0 for us already: https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html